JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/listener/BaseOntologyModelManager.java

```diff
@@ -83,7 +83,7 @@ public static void saveToOwl(OntModel jenaOwlModel, String iriOfChimney, String
         } else {
             filePath2= Paths.get(ABSDIR_KB_TEST,"ships",mmsi,"Chimney-1.owl").toString();
         }
-        logger.info("the filepath created= "+filePath2);
+        LOGGER.info("the filepath created= "+filePath2);

```

Comment:
Agreed. I've updated the JPS Base Library to use Log4j2, but not improved the quality/coverage of the logging statements; this will be up to the individual developers.